### PR TITLE
Update to several pages

### PR DESCRIPTION
**Pull request by gregorydlogan**
Updated
 - Getting started (cribbed liberally from Sakai)
 - The "Try It!" page which gets included everywhere
 - The FAQ (cribbed liberally from several other Apereo projects)

### CODE REVIEW

**Comment by gregorydlogan** [comment]
Github tells me that this is rendered at https://gregorydlogan.github.io/opencast.github.io/ for those wanting to see it in action.

**Comment by gregorydlogan** [comment]
Added:
- Releases page

File: software.md [hunk 1]
```diff
@@ -4,34 +4,13 @@ description: Opencast is an open source solution for automated video capture and
...
-{% endif %}
-{% endfor %}
-
----
 <!-- Lizenzen unsplash.com: https://unsplash.com/license -->
 
 {% include fullsizebox.html
 title="Install Opencast"
-description="Here you will find a test server, installation instructions and the general documenation for Opencast. [Read more…](install)"
+description="Here you will find a test server, installation instructions and the general documenation for Opencast. [Read more…](../releases)"
```
Comment (rrolf) [inline software.md]: Don't use ../releases that would link to the wrong page, at least in your fork.

File: software.md [hunk 2]
```diff
@@ -4,34 +4,13 @@ description: Opencast is an open source solution for automated video capture and
...
 image="assets/img/software_install.png"
-linkurl="software"
+linkurl="../releases"
```
Comment (rrolf) [inline software.md]: same here.

**Pull request merged**